{
"input": {
"_geometry": "vecgeom",
"_instance": 0,
"_name": [
"simple-cms+field+msc",
"vecgeom",
"gpu"
],
"_num_events": 7,
"_num_primaries": 9100,
"_outdir": "simple-cms+field+msc-vecgeom-gpu",
"_timeout": 600.0
},
"stderr": [
"status: Loading input and initializing problem data",
"status: Initializing Geant4",
"info: Loading geometry from /home/s3j/celeritas-regression/input/simple-cms.gdml",
"info: Loading Geant4 geometry from GDML at /home/s3j/celeritas-regression/input/simple-cms.gdml",
"status: Initializing tracking information",
"status: Reading and building Seltzer Berger model data",
"status: Reading and building Livermore PE model data",
"status: Transporting",
"terminate called after throwing an instance of 'celeritas::RuntimeError'",
"  what():  /home/s3j/.local/src/celeritas/src/corecel/data/DeviceAllocation.cc:61:",
"celeritas: cuda error: cudaFree(ptr) failed:",
"    unspecified launch failure",
"",
"### CAUGHT SIGNAL: 6 ### address: 0x5a950027b471,  signal =  SIGABRT, value =    6, description = abort program (formerly SIGIOT). ",
"",
"Backtrace:",
"[PID=2602097, TID=-1][ 0/15]> /lib64/libc.so.6(gsignal+0x10f) [0x7fec2978237f]",
"[PID=2602097, TID=-1][ 1/15]> /lib64/libc.so.6(abort+0x127) [0x7fec2976cdb5]",
"[PID=2602097, TID=-1][ 2/15]> /lib64/libstdc++.so.6(+0x9009b) [0x7fec2a13a09b]",
"[PID=2602097, TID=-1][ 3/15]> /lib64/libstdc++.so.6(+0x9653c) [0x7fec2a14053c]",
"[PID=2602097, TID=-1][ 4/15]> /lib64/libstdc++.so.6(+0x95559) [0x7fec2a13f559]",
"[PID=2602097, TID=-1][ 5/15]> /lib64/libstdc++.so.6(__gxx_personality_v0+0x2a8) [0x7fec2a13fed8]",
"[PID=2602097, TID=-1][ 6/15]> /lib64/libgcc_s.so.1(+0x10b03) [0x7fec29b20b03]",
"[PID=2602097, TID=-1][ 7/15]> /lib64/libgcc_s.so.1(_Unwind_RaiseException+0x2b1) [0x7fec29b21071]",
"[PID=2602097, TID=-1][ 8/15]> /lib64/libstdc++.so.6(__cxa_throw+0x3b) [0x7fec2a1407eb]",
"[PID=2602097, TID=-1][ 9/15]> /home/s3j/.local/src/celeritas/build-reldeb-vecgeom/src/libceleritas.so(+0x6ab04) [0x7fec2d1cdb04]",
"[PID=2602097, TID=-1][10/15]> /home/s3j/.local/src/celeritas/build-reldeb-vecgeom/src/libceleritas.so(_ZN9celeritas7StepperILNS_8MemSpaceE1EED2Ev+0x2f) [0x7fec2d2a51bf]",
"[PID=2602097, TID=-1][11/15]> /home/s3j/.local/src/celeritas/build-reldeb-vecgeom/app/demo-loop() [0x43c154]",
"[PID=2602097, TID=-1][12/15]> /home/s3j/.local/src/celeritas/build-reldeb-vecgeom/app/demo-loop() [0x41461e]",
"[PID=2602097, TID=-1][13/15]> /lib64/libc.so.6(__libc_start_main+0xf3) [0x7fec2976e493]",
"[PID=2602097, TID=-1][14/15]> /home/s3j/.local/src/celeritas/build-reldeb-vecgeom/app/demo-loop() [0x414e2e]",
"",
": Aborted (Signal sent by tkill() 2602097 23189)"
],
"stdout": [
"/home/s3j/.local/src/celeritas/src/celeritas/global/alongstep/detail/EnergyLossApplier.hh:75:",
"celeritas: internal assertion failed: step_limit->action != track.boundary_action()"
]
}